lib/node_modules/@stdlib/iter/cunone/package.json

```diff
@@ -0,0 +1,85 @@
+{
+    "name": "@stdlib/iter-cunone",
```

Comment:
Package name is incorrect.

Reply:
Is there a specific naming convention for this type of iterator that's not being followed?
here a different expected name for this package?
2 @kgryte

Reply:
Yes, that is correct. You can confirm this by examining all other packages in the main project repository. In short, package names should match the directory path.